torch/_torch_docs.py

```diff
@@ -2563,11 +2563,21 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.dot,
            r"""
-dot(input, tensor) -> Tensor
+dot(input, other, *, out=None) -> Tensor
 
-Computes the dot product (inner product) of two tensors.
+Computes the dot product (inner product) of two 1D tensors.
```

Comment:
I'd kill the "inner product" reference since there are multiple possible inner products that can be defined on R^n.